### DISCUSSION TITLE
Home Assistant OS VM

### DISCUSSION THREAD

**Discussion by Tyrant82** [tteck/Proxmox#189] [Q&A] [answered]
Hello.

How can i update the HAOS-VM 8.0rc1 to rc2 please?

br

**Comment by tteck** [comment]
I don't use HAOS, but...
I think there should be an update button in the supervisor section

**Comment by Tyrant82** [comment]
![haos-vm](https://user-images.githubusercontent.com/4549691/164950499-ade752ca-381e-4259-8f0a-94906c7dc0c0.png)
I uploaded a screenshot, but there is no update button (yet?)

**Comment by tteck** [comment]
What's in the 3 dots (hamburger) on the far right?

**Reply by Tyrant82** [reply]
Hardware
Import from USB
Move volume

**Comment by tteck** [comment, marked as answer]
Maybe someone with HAOS knowledge will chime in.

**Reply by mschabhuettl** [reply]
Go in Proxmox Container to Console:

$ ha os update
$ ha core update

**Reply by Tyrant82** [reply]
I will try that.

Thank you/Dankeschön.

**Reply by Tyrant82** [reply]
So os update installed my HAOS from 8rc1 to Home Assistant OS 7.6 .. that didnt go as i wanted lol..

**Reply by mschabhuettl** [reply]
@Tyrant82 are you on beta channel in supervisor?

As I did os update (while running supervisor beta-channel), HASSOS updated from 7.6 to 8rc1.
Sorry!

Edit1: supervisor beta not core

**Reply by Tyrant82** [reply]
No problem :) I activated the beta channel in the supervisor and now HAOS mentioned there is a new 8rc2 available and updated to it.

**Reply by mschabhuettl** [reply]
nice ;-)
